core/magisk_module/post-fs-data.sh

```diff
@@ -20,5 +20,7 @@
 
 MODDIR=${0%/*}
 
+setenforce 0
+
```

Comment:
？

Reply:
two reasons:
1. Companion is not yet working so we can only set permissive to load dex
2. Debuggerd needs to load the backtrace